app/build.gradle

```diff
@@ -108,8 +108,7 @@ dependencies {
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.4'
 
     implementation 'androidx.preference:preference:1.1.0'
-    implementation "com.mikepenz:aboutlibraries-core:v8.0.0-rc03"
-    implementation "com.mikepenz:aboutlibraries:7.1.0"
+    implementation "com.mikepenz:aboutlibraries:8.1.0"
```

Comment:
If this commit's build successes, I will definitely agree with your commit. Well, nice! https://github.com/mikepenz/AboutLibraries/commit/01350cb43c4169ab5e01d4ab9507dfdd3363e376

Reply:
Yes, there were some issue with `aboutlibraries`, but I was able to figure it out (basically, it required the new declaration in `AndroidManifest.xml`).